src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -359,8 +347,8 @@ class WatershedFilterAlgo : public ITKFilter
 	virtual std::string get_layer_prefix() const
```

Comment:
You can certainly keep this as part of your fork. The layer name is an editable field, so keeping track of parameters in provenance is the correct solution. Layer names should be as minimal as possible.

Ayla

> On Jul 1, 2016, at 8:14 PM, Braxton <notifications@github.com> wrote:
> 
> In src/Application/Filters/Actions/ActionWatershedFilter.cc <https://github.com/SCIInstitute/Seg3D/pull/161#discussion_r69370981>:
> 
> > @@ -359,8 +347,8 @@ class WatershedFilterAlgo : public ITKFilter
> >  	virtual std::string get_layer_prefix() const
> Can I keep it as a selectable option? I find it useful for quickly comparing the watersheds.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/SCIInstitute/Seg3D/pull/161/files/ac9d5b2fba17d4584d106c0a5fbaa0287b7b94c8..c5d884cc49746da3b8976489652bdf940ea0af76#r69370981>, or mute the thread <https://github.com/notifications/unsubscribe/ABVaUYIeGlSkOjJzDkhK_244ijj5YJo3ks5qRcmZgaJpZM4I8ehi>.
>